scripts/installer.nsh

```diff
@@ -0,0 +1,8 @@
+# This macro fetches the currently install MM version via MSI (if any) and uninstalls it first
+!macro customInit
```

Comment:
This is an NSIS script macro for when auto-update occurs. The user will be switched over to the EXE installation method, so we need to remove the MSI.

This should only be for users who've installed on a machine they control, on a per-user basis. Auto-update is automatically **disabled** for per-machine installations.